docs/changelog.rst

```diff
@@ -338,6 +338,8 @@ Fixes:
 * :doc:`/plugins/web`: DELETE and PATCH methods are disallowed by default.
   Set ``readonly: no`` web config option to enable them.
   :bug:`3870`
+* :doc:`/plugins/discogs`: Remove step to encode to ascii
```

Comment:
Maybe something slightly more descriptive of the fix: "Avoid a crash when searching for music with non-ASCII metadata."?

Reply:
I don't think the crash was because the content of the query, but that it was being passed around as `bytes` datatype... 

... without having looked deeper into it I suspect the old discogs client still accepted that, but the new one (that was added a few weeks ago in https://github.com/beetbox/beets/commit/00252ab28ffc7d8e25cd5e420b6c086bfc178023) didn't?

Happy to change the text though to something else... just not sure what...

Reply:
Got it! The new library would indeed explain it. Maybe we don't need a changelog at all, then, because this can essentially be folded into the change to use the new library.

Reply:
Ok, sure! I'll remove it.

Reply:
... ok I just removed that last commit (force pushed it back one commit...).